test-jmeter/pom.xml

```diff
@@ -0,0 +1,140 @@
+                                <filesets>
+                                    <fileset>
+                                        <directory>target</directory>
+                                        <includes>
+                                            <include>jmeter/reports/**/*</include>
+                                        </includes>
+                                    </fileset>
+                                </filesets>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+
+
+
+                <plugin>
+                    <groupId>com.lazerycode.jmeter</groupId>
+                    <artifactId>jmeter-maven-plugin</artifactId>
+                    <executions>
+                        <!-- Generate JMeter configuration -->
+                        <execution>
+                            <id>configuration</id>
+                            <goals>
+                                <goal>configure</goal>
+                            </goals>
+                        </execution>
+                        <!-- Run JMeter tests -->
+                        <execution>
+                            <id>jmeter-tests</id>
+                            <goals>
+                                <goal>jmeter</goal>
+                            </goals>
+                        </execution>
+                        <!-- Fail build on errors in test -->
+                        <execution>
+                            <id>jmeter-check-results</id>
+                            <goals>
+                                <goal>results</goal>
+                            </goals>
+                        </execution>
+
+                    </executions>
+                    <configuration>
+                        <testFilesDirectory>${project.basedir}/src/test/jmeter</testFilesDirectory>
+                        <testResultsTimestamp>true</testResultsTimestamp>
+
+                        <propertiesUser>
+                            <jmeter.save.saveservice.output_format>csv</jmeter.save.saveservice.output_format>
+                            <CookieManager.save.cookies>true</CookieManager.save.cookies>
+                            <sample_variables>JVM_HOSTNAME,SESSION_ID,X_DOT_SERVER</sample_variables>
+                            <resultcollector.action_if_file_exists>DELETE</resultcollector.action_if_file_exists>
+                            <host>${jmeter.host}</host>
+                            <port>${jmeter.port}</port>
+                            <rampup>${jmeter.rampup}</rampup>
+                            <startup.delay>${jmeter.startup.delay}</startup.delay>
+                            <test.duration>${jmeter.test.duration}</test.duration>
+                            <thread.number>${jmeter.thread.number}</thread.number>
+                        </propertiesUser>
+                        <!-- <jMeterProcessJVMSettings>
```

Comment:
Should this code be commented?

Reply:
We currently do not know what the best memory settings are so I left them unset for the time being.   I left this here as a reminder to come back to this, but adding commented code like this is not good practice and would have flagged with sonar.   I have decided to remove this and clarify the Readme on memory configuration